cli/args/flags.rs

```diff
@@ -480,10 +480,9 @@ static ENV_VARIABLES_HELP: &str = r#"ENVIRONMENT VARIABLES:
     DENO_NO_PROMPT       Set to disable permission prompts on access
                          (alternative to passing --no-prompt on invocation)
     DENO_WEBGPU_TRACE    Directory to use for wgpu traces
-    DENO_JOBS            Number of parallel workers used for test subcommand.
-                         Defaults to number of available CPUs when used with
-                         --jobs flag and no value is provided.
-                         Defaults to 1 when --jobs flag is not used.
+    DENO_JOBS            Number of parallel workers used for the --parallel
```

Comment:
I thought about renaming this, but `DENO_MAX_PARALLELISM` is long. Additionally, cargo uses `CARGO_JOBS` for this env variable, so I think it makes sense? We can also reuse this env var for deno fmt and lint in the future. Thoughts?

Reply:
Sounds good to me 👍